.github/RELEASE.md

```diff
@@ -5,12 +5,15 @@ This is a guide to know the steps to create a new release.
 1. Update the version in [BASHUNIT_VERSION](../bashunit)
 1. Update the version in [CHANGELOG.md](../CHANGELOG.md)
 1. Update the version in [package.json](../package.json)
+1. Update the checksum(sha256) in [package.json](../package.json)
```

Comment:
Would it be possible to create a script that automatically makes the checksum (and other possible manual steps)?

Reply:
well, the script is just in the line below :)
if you mean, creating a script to automate this whole process, yeah, that would be a next step